Automattic/woocommerce-payments
# Add E2E tests for merchant multi-currency setup

Closes #7382 

#### Changes proposed in this Pull Request

Implement E2E tests for multi-currency setup functionality, including:
- Activate multi-currency support.
- Deactivate multi-currency support.
- Change single currency rate, price rounding, and charm pricing.
- Currency shows the correct number of decimal points.

#### Testing instructions

1. Set up a clean E2E local environment.
2. Execute `npm run test:e2e merchant-multi-currency-setup.spec.js`.
3. Confirm that the tests pass in your local environment.
4. Verify that all E2E tests pass in the GitHub (GH) checks.

-------------------


**Post merge**
